man/man1/pint.1

```diff
@@ -202,6 +202,13 @@ option is only valid when used with the
 .I servers
 argument. It provides information about the SMT (update) servers operated
 and maintained by SUSE.
+.IP "--bz2"
+This option  uses compression type bz2 to compress the data over the network. This option is mutually exclusive with the
+.I --gzip 
+and
+.I --xz
+option. bz2 is the default option if neither of mantioned three compression 
```

Comment:
Grammatically `neither` should be used with when there are two options, and `none` when there are more the two options. Also `mantioned` is presumably meant to be `mentioned`, but is superfluous, and can be replaces with a `the`:

```suggestion
option. bz2 is the default option if none of the three compression 
```